cli_tools_e2e_test/gce_image_import_export/test_suites/import/import_tests.go

```diff
@@ -147,6 +147,9 @@ func (t testCase) runPostTranslateTest(ctx context.Context, imagePath string,
 		"image_under_test": {
 			Value: imagePath,
 		},
+		"startup_script": {
```

Comment:
why this? I thought you only add new test suite

Reply:
I reused the workflow file that the import test uses, since the onestep test is also under gce_image_import_export. The only change in the workflow file is to supply which start up script to use, so the import test is changed to also supply the start up script file path.